provisionersdk/scripts/bootstrap_linux.sh

```diff
@@ -43,6 +43,43 @@ if ! chmod +x $BINARY_NAME; then
 	exit 1
 fi
 
+haslibcap2() {
+	command -v setcap /dev/null 2>&1
+	command -v capsh /dev/null 2>&1
+}
+
+# Attempt to add CAP_NET_ADMIN to the agent binary. This allows us to increase
+# network buffers which improves network transfer speeds.
+if [ -n "${USE_CAP_NET_ADMIN}" ]; then
```

Comment:
The templates don't set this env var so this never gets hit right?

Reply:
Yeah, I'll add that.